tests/cli/test_utils.py

```diff
@@ -355,3 +356,18 @@ def test_validate_assistant_id_in_config_preserves_comment() -> None:
 
     # reset input files to original state
     rasa.shared.utils.io.write_yaml(original_config_data, config_file, True)
+
+
+@pytest.mark.parametrize(
+    "text_input, button",
+    [
+        ("hi this is test text\n", "hi this is test text"),
+        ("hi this is test text (/button_one)", "/button_one"),
+        ("hi this is test text (and something) (/button_one)", "/button_one"),
```

Comment:
Is this how the button gets represented from its yaml presentation? How does it end up in this string format? Just curious and don't have time to walk through the code 😅 
```
- title: "some text (blabla)"
   payload: /some_intent
```

Reply:
When someone selects the button in cli and presses enter it just adds the button after the title in text like `some text (blabla) (/some_intent)` but if it's free text it's `string + '\n'`, there is an example in the jira ticket.